src/REPL/replxx/Util.cpp

```diff
@@ -0,0 +1,133 @@
+#include <Ark/REPL/replxx/Util.hpp>
+#include <cstring>
+#include <regex>
+
+int utf8str_codepoint_len(char const *s, int utf8len) 
+{
+    int codepointLen = 0;
+    unsigned char m4 = 128 + 64 + 32 + 16;
+    unsigned char m3 = 128 + 64 + 32;
+    unsigned char m2 = 128 + 64;
+
+    for(int i = 0; i < utf8len; ++ i, ++ codepointLen)
+    {
+        char c = s[i];
+
+        if ((c & m4) == m4)
+        {
+            i += 3;
+        }
+        else if((c & m3) == m3)
+        {
+            i += 2;
+        }
+        else if(( c & m2 ) == m2)
+        {
+            i += 1;
+        }
```

Comment:
mostly coding style, since there is only one instruction per if, the {} can be removed